.github/workflows/on-pull-request.yml

```diff
@@ -38,6 +38,22 @@ jobs:
         run: brm validate
         working-directory: ${{ needs.get-module-to-validate.outputs.module_dir }}
 
+  shellcheck:
+    name: runner / shellcheck
+    runs-on: ubuntu-latest
+    needs: get-module-to-validate
+    if: needs.get-module-to-validate.outputs.module_dir
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review # Change reporter.
+          path: ${{ needs.get-module-to-validate.outputs.module_dir }}
+          pattern: "*.sh"
```

Comment:
Should we only check shell scripts under the changed module folder?

Reply:
I really like this tool when I am writing shell scripts. I'd recommend using it for a shell script wherever it might be.

Reply:
In that case, do we want the step be part of the `validate-non-module-files` job? Currently, it will only be invoked if a module is changed.